locations/k9.yml

```diff
@@ -21,6 +21,31 @@ hosts:
     role: ap
     model: "aruba_ap-303"
     wireless_profile: freifunk_default
+  - hostname: k9-ap-iuljan
+    role: ap
+    model: "dlink_dap-x1860-a1"
+    wireless_profile: freifunk_default
+  - hostname: k9-switch-front-house
+    role: ap #??
+    model: "ubnt_usw-flex"
+    poemgr_ports:
+      - name: lan2
+        port: 3
+        disabled: 1
+      - name: lan3  # only PoE here
+        port: 2
+        disabled: 0
+      - name: lan4
+        port: 1
+        disabled: 1
+      - name: lan5
+        port: 0
+        disabled: 1
+
+    poemgr_power_budget: 24 #???
```

Comment:
This setting is only required if you use active to passive converters that negotiate more power than they actually draw and therefore devices get not powered properly. With PoE only on one port you should not need this at all.